frontend/simple/views/containers/contributions/IncomeForm.vue

```diff
@@ -0,0 +1,245 @@
+<template>
+  <modal ref="modal" :isActive="true" @close="cancelForm">
+    <form class="modal-card-content c-wrapper" @submit.prevent="verifyForm" novalidate="true">
+      <i18n tag="h2" class="title is-3">Income Details</i18n>
+      <div class="columns is-mobile c-wrapper-columns">
+        <div class="column c-form">
+          <fieldset class="fieldset">
+            <i18n tag="legend" class="sr-only">Income details</i18n>
+
+            <div class="field is-narrow gi-fieldGroup">
+              <i18n tag="p" class="label">Do you make at least {{fakeStore.mincomeFormatted}} per month?</i18n>
+              <i18n tag="strong" class="help has-text-danger has-text-weight-normal gi-help" v-if="ephemeral.formVerified && $v.form.option.$invalid">{{infoRequired}}</i18n>
+              <div class="control">
+                <label class="gi-tick">
+                  <input class="gi-tick-input" type="radio" value="no" v-model="form.option" @change="resetFormVerify">
+                  <span class="gi-tick-custom"></span>
+                  <i18n>No, I don't</i18n>
+                </label>
+                <label class="gi-tick">
+                  <input class="gi-tick-input" type="radio" value="yes" v-model="form.option" @change="resetFormVerify">
```

Comment:
Both here, above, and elsewhere, should you perhaps bind to `$v.form.option.$modal` so that [`$dirty` is automatically set](https://monterail.github.io/vuelidate/#sub-v-values), and so that you don't need to call `resetFormVerify`?

Reply:
Does this comment still makes sense? :/